core/src/main/resources/jenkins/security/ResourceDomainConfiguration/help-url.html

```diff
@@ -0,0 +1,41 @@
+<p>
+    Jenkins serves many files that are potentially created by users that are not fully trusted, such as files in project workspaces or archived artifacts.
+    Without this option set, Jenkins will only serve these files with the HTTP header <code>Content-Security-Policy</code> ("CSP") set to a value disallowing many modern web features to prevent cross-site scripting (XSS) attacks on Jenkins users accessing these files.
+    While the specific value for the CSP header is user configurable (and can even be disabled), doing so is a trade-off between security and functionality.
+</p>
+<p>
+    If this option is set, Jenkins will redirect requests for user-created resource files to URLs starting with the URL configured here.
```

Comment:
```suggestion
    If a resource domain is configured, Jenkins will redirect requests for user-created resource files to URLs starting with the URL configured here.
```
A very minor concern, but I'm going for a parallelism with my earlier proposal and being specific about which option we're talking about -- not the CSP header option.